File: c++_backup/Amlyan/LCM.cpp
#include<iostream>
using namespace std;
// to return the LCM of a number
int main()
{
	int a,b,i;
	cout<<"\n Enter two numbers :\n"<<endl;
	cin>>a;
	cin>>b;
	for(i=1;i<=b;i++)
	{
		if(a*i%b == 0)
		{
			cout<<"LCM = "<<a*i<<endl;
			break;
		}
	}
	return 0;
}

File: java_backup/my java/jatin/lcm.java

class lcm
    {
        int a, b, k;
        void input(int a,int b)
            {
                k=lcm(a, b);
                System.out.print("lcm" +k);
            }
            int lcm(int a,int b)
            {
                 
            int i=1;
            while((a*i)%b !=0)
            i++;
            return(a*i);
        }
    }
